### DISCUSSION TITLE
Change (override) sidebar layout buttons

### DISCUSSION THREAD

**Discussion by StrayTheWolf** [orchidsoftware/platform#1875] [Q&A] [answered]
Hello everyone, tell me please, there is a task to change the style and appearance of the standard side menu buttons in
Menu :: make in file
PlatformProvider.
I am not clear where and how to change the template in which the "li" elements are generated.
My idea was to add new css classes to the standard ones that Orchid generates

**Comment by qnts** [comment, marked as answer]
Menu view is located in `/vendor/orchid/platform/resources/views/actions/menu.blade.php`.

If you want to customize this view, copy it to your app views dir as `/resources/views/vendor/platform/actions/menu.blade.php` and modify as you wish.

**Reply by StrayTheWolf** [reply]
Thanks a lot
